### PR TITLE
Edit Data from queue, remove delete button

### PR DESCRIPTION
**Pull request by bdonald25**
#69 
@stowellm

### CODE REVIEW

File: Libs/iSENSE Imports/src/edu/uml/cs/isense/queue/QueueAlter.java [hunk 1]
```diff
@@ -37,13 +37,12 @@ public void onCreate(Bundle savedInstanceState) {
...
 		changeData = (Button) findViewById(R.id.queuealter_change_data);
 		selectProj = (Button) findViewById(R.id.queuealter_choose_proj);
-		delete     = (Button) findViewById(R.id.queuealter_delete     );
 		cancel     = (Button) findViewById(R.id.queuealter_cancel     );
 
 		Bundle extras = getIntent().getExtras();
 
 		if (extras != null) {
-			boolean isAlterable = false; //extras.getBoolean(IS_ALTERABLE); TODO editing data is broken
+			boolean isAlterable = extras.getBoolean(IS_ALTERABLE); //TODO editing data is broken
```
Comment (stowellm) [inline Libs/iSENSE Imports/src/edu/uml/cs/isense/queue/QueueAlter.java]: Is editing data still broken? I thought you fixed it.

File: Libs/iSENSE Imports/src/edu/uml/cs/isense/queue/QueueEditData.java [hunk 2]
```diff
@@ -80,62 +90,97 @@ public void onClick(View v) {
...
 	private void fillScrollView() {
-		
+
 		int i = 0;
-		String rawFieldData = alter.getData().replace("[", "").replace("]", "").replace("\"", "");
+		String rawFieldData = alter.getData().replace("[", "").replace("]", "").replace("\"", "").replace("\\", "");
 		String[] fieldData = rawFieldData.split(",");
-		
+
+        Log.e("data: ", rawFieldData);//total data
```
Comment (stowellm) [inline Libs/iSENSE Imports/src/edu/uml/cs/isense/queue/QueueEditData.java]: Is that.. a log :gasp:

File: Libs/iSENSE Imports/src/edu/uml/cs/isense/queue/QueueEditData.java [hunk 3]
```diff
@@ -80,62 +90,97 @@ public void onClick(View v) {
...
-			EditText dataText = (EditText) v.findViewById(R.id.edit_row_text);
-			if (dataText.getText().toString().length() != 0)
-				row.put(dataText.getText().toString());
-			else
-				row.put(" ");
-		}
-		
-		data.put(row);
-		
+		int max = editDataList.getChildCount();//all data edit texts + field labels
```
Comment (stowellm) [inline Libs/iSENSE Imports/src/edu/uml/cs/isense/queue/QueueEditData.java]: `max` looks like an unused variable

**Comment by bdonald25** [comment]
@stowellm

**Comment by stowellm** [comment]
Good to pull :hamburger:

**Pull request merged**